translate/tools/test_podebug.py

```diff
@@ -28,8 +28,8 @@ class TestPODebug:
     debug = podebug.podebug()
 
     def setup_method(self, method):
-        self.postore = po.pofile(PO_DOC)
-        self.xliffstore = xliff.xlifffile(XLIFF_DOC)
+        self.postore = po.pofile(PO_DOC.encode())
```

Comment:
I would prefer that. Tests are something that should use clean and readable code so it is easy to understand and spot errors.